examples/k8s_tensorflow/README.md

```diff
@@ -0,0 +1,478 @@
+# A tensorflow on k8s example
+
+## Requirements:
+
+## pull tensorflow image:
+```shell
+[xuerq@bogon train]$ docker pull tensorflow/tensorflow
+064f9af02539: Already exists 
+390957b2f4f0: Already exists 
+cee0974db2b8: Already exists 
+ff4090f99abc: Pull complete 
+70d51ddf7c95: Pull complete 
+da76ab5d6dff: Pull complete 
+46d4527e85d3: Pull complete 
+528276ea4b2d: Pull complete 
+709fc41158c6: Pull complete 
+d63463802d36: Pull complete 
+b4c3589c6b3a: Pull complete 
+Digest: sha256:0635a564b59ac45e9a44d7c38efee9bf6c9567150236e064d8e382fbbf54fe74
+Status: Downloaded newer image for tensorflow/tensorflow:latest
+```
+## login harbor:
```

Comment:
这里是不是应该解释一下为什么要把images push到harbor上？比如

> Because the Kubernetes cluster cannot access the Internet, we need to push Tensorflow images onto the private Docker image service.